packages/components/accordion/src/AccordionPanel/AccordionPanel.tsx

```diff
@@ -16,7 +16,7 @@ export interface AccordionPanelProps {
   /**
    * An unique id that is necessary for the aria roles and properties
    */
-  ariaId: number;
+  ariaId: string;
 }
 
 export const AccordionPanel: FC<AccordionPanelProps> = ({
```

Comment:
it would be nice to add ref

Reply:
could you elaborate, why do we need a ref? like, exposing a ref or passing down a ref?

Reply:
`React.forwardRef` and then assign ref to the top element in JSX